src/main/java/sipay/responses/ecommerce/Refund.java

```diff
@@ -20,7 +20,7 @@ public Refund(JSONObject resp) {
         super(resp);
 
         JSONObject payload = ((JSONObject) resp.opt("payload"));
-        if (payload.length() > 0) {
+        if (payload != null) {
```

Comment:
Also catching the exception in the send method, so we can have the original trace from HTTP. Otherwise we are masking the original error and it will be more difficult to debug